src/tests/test_libs_xml_utils.py

```diff
@@ -262,7 +288,7 @@ def test_fail_included_cert_not_from_a_known_ca(self) -> None:
             verify_xml_signature(xml_doc, trusted_x509_cert=None)
         self.assertEqual(
             cm.exception.args,
-            ('unable to get local issuer certificate',),
+            ('validation failed: cert is not valid at validation time',),
```

Comment:
The new error message doesn't seem related to the test's purpose:

> Without cert: fails because the issuer of the cert in the signature is not a known CA.

Reply:
I generated a new xml file with a not expired cert, I got this error message 
```
('validation failed: Certificate is missing required extension',),
```
but with an EE certificate instead of a `basicConstraints.cA` because otherwise I was geeting this error 

```
validation failed: basicConstraints.cA must not be asserted in an EE certificate
```

Reply:
The objective of the test is to check the behavior of `verify_xml_signature()` when the certificate contained in the XML document is not trusted (because `trusted_x509_cert` is `None`). The expected behavior is that the signature validation fails because SignXML is unable to find a trusted certificate for the issuer in its local certificate authority store. We must be sure that we are still testing the same thing.

Reply:
See https://github.com/cordada/lib-cl-sii-python/pull/736#discussion_r1881105783.